<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt4ProjectManager::Internal::MobileAppWizardMaemoOptionsPage</class>
 <widget class="QWizardPage" name="Qt4ProjectManager::Internal::MobileAppWizardMaemoOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="appIconLabel">
     <property name="text">
      <string>Application icon (64x64):</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="pngIconButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
